pom.xml

```diff
@@ -73,7 +73,7 @@
     <jackson.groupId>com.fasterxml.jackson.core</jackson.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
     <jackson.version>2.13.2</jackson.version>
-    <jackson-databind.version>${jackson.version}</jackson-databind.version>
+    <jackson-databind.version>2.13.2.2</jackson-databind.version>
```

Comment:
What is the reason that we split the two versions(jackson.version, jackson-databind.version)?

Reply:
jackson-databind has a different versioning. They released two micro-patches to resolve a CVE. [Here](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13)there is more information about the release.

Reply:
Thanks @braiscouce